scripts/db/createdb.js

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env node
+
```

Comment:
Just me or this indentation is broken?

Reply:
Yeah, this is weird, I saw it too, but the code looks good locally.

![image](https://user-images.githubusercontent.com/849872/68998790-729bc700-087c-11ea-8aa5-068c33139d57.png)

Maybe something with Github?